frontend/app/scripts/services/fixindepnet.js

```diff
@@ -18,7 +18,7 @@
       var fixField = function(value) {
          return value.replace('https://forge.indepnet.net', 'https://forge.glpi-project.org');
       };
-      var fields = ['logo_url', 'readme_url', 'download_url', 'homepage_url'];
+      var fields = ['logo_url', 'readme_url', 'download_url', 'homepage_url', 'changelog_url'];
```

Comment:
This one will not be required. It was made only for old URLs.
```suggestion
      var fields = ['logo_url', 'readme_url', 'download_url', 'homepage_url'];
```